modules/react/layout/lib/utils/stack.ts

```diff
@@ -26,50 +22,28 @@ export type StackStyleProps = {
   shouldWrapChildren?: boolean;
 };
 
-const column = (value: StackSpacing) => ({
-  marginTop: spaceTokens[value as CanvasSpaceKeys] || value,
-});
-
-const columnReverse = (value: StackSpacing) => ({
-  marginBottom: spaceTokens[value as CanvasSpaceKeys] || value,
-});
-
-const row = (value: StackSpacing, isRTL = false) => {
-  const attr = isRTL ? 'marginRight' : 'marginLeft';
-  return {
-    [attr]: spaceTokens[value as CanvasSpaceKeys] || value,
-  };
+const getSpace = (value: SystemPropValues['space']) => {
+  return spaceTokens[value as CanvasSystemPropValues['space']] || value;
 };
 
-const rowReverse = (value: StackSpacing, isRTL = false) => {
-  const attr = isRTL ? 'marginLeft' : 'marginRight';
-  return {
-    [attr]: spaceTokens[value as CanvasSpaceKeys] || value,
-  };
-};
+export function stack<P extends StackStyleProps>(props: P) {
+  const {flexDirection = 'row', spacing} = props;
+  let styles = {};
 
-const stackDirectionProps = {
-  column,
-  'column-reverse': columnReverse,
-  row,
-  'row-reverse': rowReverse,
-};
+  if (flexDirection === 'column') {
+    styles = {marginTop: getSpace(spacing)};
+  }
 
-export function stack<P extends StackStyleProps & {theme?: PartialEmotionCanvasTheme}>(props: P) {
-  let styles = {};
-  const {flexDirection = 'row', spacing} = props;
-  if (flexDirection === 'column' || flexDirection === 'column-reverse') {
-    const stackFn = stackDirectionProps[flexDirection];
-    styles = stackFn(spacing);
+  if (flexDirection === 'column-reverse') {
+    styles = {marginBottom: getSpace(spacing)};
+  }
+
+  if (flexDirection === 'row') {
+    styles = {marginInlineStart: getSpace(spacing)};
```

Comment:
Because we dropped IE11, we can use CSS Logical properties. We don't have to check the theme for direction. The browser will do it for us!